logging/bin/deploy_kibana_content.sh

```diff
@@ -183,6 +183,41 @@ fi
 ./logging/bin/import_kibana_content.sh logging/kibana/tenant          cluster_admins
 
 
+
+# create the all logs RBACs
+add_notice "**Elasticsearch/Kibana Access Controls**"
+LOGGING_DRIVER=true ./logging/bin/security_create_rbac.sh _all_ _all_
+
+# Create the 'logadm' Kibana user who can access all logs
+LOG_CREATE_LOGADM_USER=${LOG_CREATE_LOGADM_USER:-false}
+if [ "$LOG_CREATE_LOGADM_USER" == "true" ]; then
+
+   if user_exists logadm; then
+      log_warn "A user 'logadm' already exists; leaving that user as-is.  Review its definition in Kibana and update it, or create another user, as needed."
```

Comment:
"Review its definition in Kibana and update it, or create another user, as needed." seems superfluous

Reply:
The point of the additional sentence is to point out that while a user with this name already exists, it may or may not be configured the way the user expected.  We wouldn't want the person running the deployment script to interpret the fact that this user (name) already exists as "this user with all of the expected permissions has already been created".